docs/api.md

```diff
@@ -8,7 +8,7 @@ rerun the generator, to avoid loosing the changes.
 
 ## How to import
 ```javascript
-import { open, save, DB, Empty } from "https://deno.land/x/sqlite/mod.ts"
+import { open, save, DB, SqliteError, Empty } from "https://deno.land/x/sqlite/mod.ts"
```

Comment:
This is not your fault, but there was a bug in the doc generator which was corrected after your branch. Could you rebase on master to get the fix and rerun the generator? Apologies for this inconvenience 😅

Reply:
No problem. :slightly_smiling_face:  Okay I rebased from master and reran the generator.